k8s/overlays/test/fixture.json

```diff
@@ -0,0 +1,3 @@
alue":{"stringValue":"go"}},{"key":"telemetry.sdk.name","value":{"stringValue":"opentelemetry"}},{"key":"telemetry.sdk.version","value":{"stringValue":"1.8.0"}}]},"scopeSpans":[{"scope":{"name":"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp","version":"semver:0.33.0"},"spans":[{"traceId":"c2824081ecac96c496ff2ab9265b9590","spanId":"894c881be709e0e5","parentSpanId":"847ab8fd2ebbc3a1","name":"HTTP GET","kind":"SPAN_KIND_CLIENT","startTimeUnixNano":"1658413468293221343","endTimeUnixNano":"1658413468359589936","attributes":[{"key":"http.method","value":{"stringValue":"GET"}},{"key":"http.url","value":{"stringValue":"http://0.0.0.0:7080/hello"}},{"key":"http.scheme","value":{"stringValue":"http"}},{"key":"http.host","value":{"stringValue":"0.0.0.0:7080"}},{"key":"http.flavor","value":{"stringValue":"1.1"}},{"key":"http.status_code","value":{"intValue":"200"}}],"status":{}},{"traceId":"7f014cc852bd0f881fa1cfd745d183b6","spanId":"5a8bd3205d2e964d","parentSpanId":"f9b36c6bc9da2da4","name":"HTTP GET","kind":"SPAN_KIND_CLIENT","startTimeUnixNano":"1658413469360430339","endTimeUnixNano":"1658413469524889310","attributes":[{"key":"http.method","value":{"stringValue":"GET"}},{"key":"http.url","value":{"stringValue":"http://0.0.0.0:7080/hello"}},{"key":"http.scheme","value":{"stringValue":"http"}},{"key":"http.host","value":{"stringValue":"0.0.0.0:7080"}},{"key":"http.flavor","value":{"stringValue":"1.1"}},{"key":"http.status_code","value":{"intValue":"200"}}],"status":{}},{"traceId":"f28b0673f1147f72b0655b4960772a05","spanId":"59dd652ebd1fb5b3","parentSpanId":"58d2299363617187","name":"HTTP GET","kind":"SPAN_KIND_CLIENT","startTimeUnixNano":"1658413470525215286","endTimeUnixNano":"1658413471213087518","attributes":[{"key":"http.method","value":{"stringValue":"GET"}},{"key":"http.url","value":{"stringValue":"http://0.0.0.0:7080/hello"}},{"key":"http.scheme","value":{"stringValue":"http"}},{"key":"http.host","value":{"stringValue":"0.0.0.0:7080"}},{"key":"http.flavor","value":{"stringValue":"1.1"}},{"key":"http.status_code","value":{"intValue":"200"}}],"status":{}},{"traceId":"01807a8911d7cc2704124128b4d9741a","spanId":"83fc30eb9796c970","parentSpanId":"1783176e15b37558","name":"HTTP GET","kind":"SPAN_KIND_CLIENT","startTimeUnixNano":"1658413472214074826","endTimeUnixNano":"1658413473129784936","attributes":[{"key":"http.method","value":{"stringValue":"GET"}},{"key":"http.url","value":{"stringValue":"http://0.0.0.0:7080/hello"}},{"key":"http.scheme","value":{"stringValue":"http"}},{"key":"http.host","value":{"stringValue":"0.0.0.0:7080"}},{"key":"http.flavor","value":{"stringValue":"1.1"}},{"key":"http.status_code","value":{"intValue":"200"}}],"status":{}}]},{"scope":{"name":"demo-client-tracer"},"spans":[{"traceId":"c2824081ecac96c496ff2ab9265b9590","spanId":"847ab8fd2ebbc3a1","parentSpanId":"","name":"ExecuteRequest","kind":"SPAN_KIND_INTERNAL","startTimeUnixNano":"1658413468293162902","endTimeUnixNano":"1658413468359675892","status":{}}]}]}]}
```

Comment:
Suggestion from our discussion.  Maybe the output of this could be the input for our fixture tests in the collector repo?

Reply:
Yup I love that idea. Right now all the config is really testing is resource detection, so I stripped down the input fixture to remove any of that. We'll have to see how close we can get this output to our collector inputs, but I think that would complete the pipeline and we should keep that in mind.